carbon-design-system/carbon
# chore: add codesandbox examples to the project

Closes #10756 

This PR adds in two codesandbox examples for `@carbon/react` and `@carbon/styles` that can be used to recreate bugs. They are currently powered by Vite versus Create React App to see if it can provide faster compilation times and an overall better experience in codesandbox

#### Changelog

**New**

- `examples` directory with `codesandbox` and `codesandbox-styles` projects

**Changed**

- Update workspaces config to include examples

**Removed**

#### Testing / Reviewing

- Pull down PR
- Run yarn to make sure everything is wired up
- Checkout each example and run `yarn develop` and verify that it renders a view with a button in it that loads the correct CSS
